Test/ServerTelemetryChannel.Test/Shared.Tests/Helpers/StubTelemetrySerializer.cs

```diff
@@ -9,7 +9,7 @@ internal class StubTelemetrySerializer : TelemetrySerializer
     {
         public Action<IEnumerable<ITelemetry>> OnSerialize = _ => { };
 
-        public override void Serialize(IEnumerable<ITelemetry> items)
+        public override void Serialize(ICollection<ITelemetry> items)
```

Comment:
The underlying method was changed, so the this override was changed to match. The underlying method was changed to allow accessing the count of items without enumerating.